### PR TITLE
Implement face_to_cell_vertices for pyramids and wedges.

### PR DESCRIPTION
**Pull request by drwells**
I need this to finish testing boundary id assignment for #10899.

### CODE REVIEW

File: include/deal.II/grid/reference_cell.h [hunk 1]
```diff
@@ -783,11 +783,39 @@ namespace ReferenceCell
         {
           AssertIndexRange(face_no, n_faces());
 
-          if (face_no == 1)
+          if (face_no == 0)
```
Comment (peterrum) [inline include/deal.II/grid/reference_cell.h:786]: Jup. This was a typo.

File: include/deal.II/grid/reference_cell.h [hunk 2]
```diff
@@ -783,11 +783,39 @@ namespace ReferenceCell
...
+#ifdef DEBUG
+          if (face == 0)
+            {
+              AssertIndexRange(vertex, 4);
+            }
+          else
+            {
+              AssertIndexRange(vertex, 3);
+            }
+#endif
```
Comment (peterrum) [inline include/deal.II/grid/reference_cell.h:807]: Why debug mode? `AssertIndexRange` is only evaluated in debug mode, isn't it?

**Review by peterrum — APPROVED** [review]
Thanks a lot, @drwells! I have checked: the tables are identical to what is contained in the pictures part of #11095.

File: include/deal.II/grid/reference_cell.h (hunk 2 above)
Comment (drwells) [reply include/deal.II/grid/reference_cell.h:807]: I can't remember if the compiler will complain about empty if-statements or not. If the CI passes then this is probably fine - I'll force push a version without the `#ifdef` check.

**Review by peterrum — APPROVED** [review]

**Pull request merged**